smoke-test-pipeline.yml

```diff
@@ -8,7 +8,7 @@ schedules:
   always: true
 
 variables:
-- group: 'spk-vg'
+- group: 'spk-vg-testing'
```

Comment:
@NathanielRose You're proposing to change the environment (via new variable group) in order to get infrastructure tests to pass but we still don't know the root cause of why we have to change the environment?

It would be nice to know exactly what the difference is here so we could add it to our documentation around integration test set up. 

Right now it seems like we are make blind changes and things _just happen_ to work

Reply:
I can revert the test for infra back to v1 until I have a solution to why the SP in the prod variable group is throwing the permissions error?